src/Engine/Loader/Loader.cpp

```diff
@@ -114,8 +114,8 @@ Texture* Loader::getTexture(const String & name) {
     return getTexture(name, GL_REPEAT, true);
 }
 
-int Loader::loadLevel(const String & name) {
-    return FileReader::loadLevel(*name.c_str());
+int Loader::loadLevel(const String & name, float ambience) {
```

Comment:
Shouldn't ambience be per material?

Reply:
yes, but we're not importing any external material properties at the moment. Until Zack does that, if he does that, i figured it was important to have one global ambience value defined in main.

Reply:
@zackragozzino